externs/shaka/player.js

```diff
@@ -1120,6 +1121,11 @@ shaka.extern.DashManifestConfiguration;
  *   As a consequence, lazy-loading media playlists won't be possible for this
  *   use case, which may result in longer video startup times.
  *   <i>Defaults to <code>false</code>.</i>
+ * @property {boolean} disableClosedCaptionsDetection
+ *   If set to false, If there is no EXT-X-MEDIA with TYPE="CLOSED-CAPTIONS" we
```

Comment:
This is worded in a confusing way. I think it'd be clearer if it started out with what happens when the value is set to true. Also I'm not sure if you need to specify that thing about NONE, since this already says that the closed captions detection won't happen if there's an EXT-X-MEDIA with TYPE="CLOSED-CAPTIONS".
I'd instead say:

```js
/**
 *   If true, disables the automatic detection of closed captions.
 *   Otherwise, in the absence of a EXT-X-MEDIA tag with TYPE="CLOSED-CAPTIONS",
 *   Shaka Player will attempt to detect captions based on the media data.
 *   <i>Defaults to <code>false</code>.</i>
 */
```

Reply:
Done!